rskj-core/src/main/java/co/rsk/peg/Bridge.java

```diff
@@ -237,6 +255,7 @@ public void init(Transaction rskTx, Block rskExecutionBlock, Repository reposito
         this.rskExecutionBlock = rskExecutionBlock;
         this.repository = repository;
         this.logs = logs;
+        this.blockchainConfig = blockchainNetConfig.getConfigForBlock(rskExecutionBlock.getNumber());
```

Comment:
If we don't need this everywhere, I'd rather do a lazy initialization.

Reply:
We actually do need it everywhere as it is used in `parseData` to check if the method is actually enabled at the current block height.